src/commands/transaction/accountlink.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ *
+ * Copyright 2018 NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import chalk from 'chalk';
+import {command, ExpectedError, metadata, option} from 'clime';
+import {AccountLinkTransaction, Deadline, TransactionHttp} from 'nem2-sdk';
+import {ProfileCommand, ProfileOptions} from '../../profile.command';
+
+export class CommandOptions extends ProfileOptions {
+    @option({
+        flag: 'p',
+        description: '<remoteaccountkey>',
+    })
+    publickey: string;
+    @option({
+        flag: 'a',
+        description: '< 0 (Link)| 1 (Unlink)>',
+    })
+    action: number;
+
+    validateAction(value: number): 0|1 {
```

Comment:
As we are returning the same value we are passing, we could write something like:

```
if([0,1].some(x => (x === value))) return value;
throw new ExpectedError('Introduce a valid action value');

```

Reply:
This looks good